src/components/moviesListMarkup/moviesListMarkup.js

```diff
@@ -5,14 +5,14 @@ import fetchTrendyMovies from '../fetchTrendyMovies/fetchTrendyMovies';
 const htmlMarkup = data =>
   data
     .map(
-      ({ backdrop_path, title, genre_ids, release_date, vote_average }) => `
-     <div class="popular-movies_item item" id="" data-modal-open>
-         <img src="${backdrop_path}" alt="" class="item__image" />
+      ({ poster_path, title, genre_ids, release_date, vote_average, id }) => `
+     <div class="item" data-id="${id}" data-modal-open>
+         <img class="item__image" src="https://image.tmdb.org/t/p/w300${poster_path}" alt=" Poster of: ${title}" />
```

Comment:
Początek ścieżki to:
https://image.tmdb.org/t/p/w500

Reply:
ścieżka posiada "/w300' zamiast 'w500', ponieważ takiego rozmiaru obraz jest wyświetlany w karcie, w500 jest za duże, za czym plik zdjęcia waży niepotrzebnie więcej niż ten z w300

Reply:
Ok. Nie było o tym mowy w dokumentacji z API bo tam jest ten adres z w500.

Reply:
Znalazłem to tutaj, patrz pod kluczem 'poster_sizes'
[https://developers.themoviedb.org/3/configuration/get-api-configuration](url)